dex/testing/dcrdex/harness.sh

```diff
@@ -59,7 +59,25 @@ cat > "./markets.json" <<EOF
 }
 EOF
 
+# Provide the postgres user password if set. 
+PG_PASS=
```

Comment:
A preferred way to do this would be to execute the script with `PG_PASS=pass ./harnes.sh` so that the file never has to be edited.